src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lot = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
```
lets use descriptive names, in java it is not acceptable to use short forms